javaslang/src/main/java/javaslang/collection/AbstractMultimap.java

```diff
@@ -223,6 +223,42 @@ public M filter(Predicate<? super Tuple2<K, V>> predicate) {
     }
 
     @Override
+    public M filter(BiPredicate<? super K, ? super V> predicate) {
+        Objects.requireNonNull(predicate, "predicate is null");
+        return filter(t -> predicate.test(t._1(), t._2()));
```

Comment:
Great, thank you!

In ideomatic Javaslang we access tuple fields directly, e.g. `t._1, t._2` instead of `t._1(), t._2()`. Could you change that please for all methods here?

The methods are useful in case of method references, e.g. `t::_1` and `t::_2`.